.circleci/config.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           name: Publish
           command: |
             sudo pip install twine
+            sudo pip3 install twine
```

Comment:
as @dchai76 said, why are you having both of `pip install` and `pip3 install` in the same job?